JaeBea/task-manager-momo
# Issue #1:  Create Dockerized environment

See issue #1 for details.

## Test Steps
1. Pull this branch and run `docker-compose up` from root.
2. Navigate to localhost:3000 and confirm acceptance criteria
3. Navigate to localhost:3100 and confirm acceptance criteria
